### PR TITLE
Poc 1 new feat: add coll-tx demo

### PR DESCRIPTION
**Pull request by ai-chen2050**
The coll-tx demo is about collecting simple utxo transactions with cops and vlc.

### CODE REVIEW

**Comment by fshif** [comment]
@ai-chen2050 resolve conflicts first

**Comment by ai-chen2050** [comment]
@fshif done

File: demos/coll-tx/src/simple_utxo.rs [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+impl SimpleUTXO {
+    /// new empty SimpleUTXO
+    pub fn new() -> Self {
+        SimpleUTXO {
+            inputs: Vec::new(),
+            outputs: Vec::new(),
+        }
+    }
+
+    /// new SimpleUTXO with inputs and outpus
```
Comment (sglk123) [inline demos/coll-tx/src/simple_utxo.rs]: misspelled word outputs

Comment (ai-chen2050) [reply demos/coll-tx/src/simple_utxo.rs]: have corrected

File: demos/coll-tx/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        let (config, handles) = start_servers(1).await;
+        // Run client
+        let mut client = Client::new(&config).await;
+
+        let tin = TxInput::new(vec![116, 104, 105, 115, 32, 105, 115, 32, 97, 32, 115, 116, 114, 105, 110, 103], 0, vec![4,5,6]);
+        let tout = TxOutput::new(100, vec![7, 8, 9]);
+        let tutxo = SimpleUTXO::new_with_inputs_outputs(vec![tin], vec![tout]);
+
+        let vec_id = tutxo.tx_id();
+        let vec_raw_tx = tutxo.encode().clone();
```
Comment (sglk123) [inline demos/coll-tx/src/lib.rs]: why clone here for fully ownership instance

**Review by sglk123 — APPROVED** [review]

File: demos/coll-tx/src/lib.rs (hunk 2 above)
Comment (ai-chen2050) [reply demos/coll-tx/src/lib.rs]: have removed

**Pull request merged**